docs/README.md

```diff
@@ -7,7 +7,27 @@ tbd
 tbd
 
 ## Getting Started
-tbd
+
+### Installation on OpenShift
+
+1. In the OpenShift Console, navigate to the OperatorHub by clicking **Operator** -> **Operator Hub** in the left side-pane.
+
+1. Search for "sail".
+
+1. Locate the sail-operator, and click to select it.
+
+1. When the prompt that discusses the community operator appears, click **Continue**.
+
+1. Verify the sail-operator is version 0.1, and click **Install**.
```

Comment:
Should we skip this step? Having it means this doc will become outdated very often.

Reply:
I mean, skip the version check.

Reply:
+1 but we should probably remove other channels to avoid confusion, we still have there 3.0-nigthly and 3.0-dp1

Reply:
I removed the step, clicking install is now part of the previous one